std/json.d

```diff
@@ -37,7 +37,7 @@ import std.traits;
 
     // check a type
     long x;
-    if (const(JSONValue)* code = "code" in j)
+    if (auto code = "code" in j)
```

Comment:
It used to be possible. I've broken libdparse in the PR mentioned previously. I have to repair this.